packages/p2p/src/components/modal-manager/modals/share-my-ads-modal/share-my-ads-card/share-my-ads-card.scss

```diff
@@ -0,0 +1,77 @@
+.share-my-ads-card {
+    background: linear-gradient(349deg, $color-grey-2 36%, var(--brand-red-coral) 25%);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-bottom: 1.5rem;
+    padding: 2rem;
+    position: relative;
+
+    @include mobile {
+        width: 90%;
+    }
+
+    &__icon {
+        height: 16px;
+        width: max-content;
+        margin: 1rem 0 3rem;
+
+        @include mobile {
+            margin-top: 0;
+        }
+    }
+
+    &__numbers {
+        display: flex;
+        flex-direction: row;
+
+        &-text {
+            display: flex;
+            flex-direction: column;
+            margin-bottom: 1rem;
+
+            &:first-child {
+                margin-right: 1rem;
+            }
+
+            &:last-child {
+                margin-bottom: 0;
+            }
+        }
+    }
+
+    &__qr {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-top: 2rem;
+
+        @include mobile {
+            margin-top: 1rem;
+        }
+
+        &-container {
+            background-color: #fff;
```

Comment:
how is this in dark mode? can we use the variable here?

Reply:
we don't really have the design for dark mode, so i kept it the same colour across both dark and light mode

Reply:
i did try using a variable previously, but then it changed in dark mode and it didn't look right hence why i just kept gave it the colour design team has given it